app/admin/user.rb

```diff
@@ -176,7 +191,9 @@ def find_resource
   #
   permit_params :full_name, :email, :institution, :role, :phone_number,
                 :is_admin,
-                :antenne_id, expert_ids: []
+                :antenne_id,
+                *User::FLAGS,
```

Comment:
`*User::FLAGS,` ça inclut tout ce qu'il y a dans `FLAGS` ?

Reply:
Yes, c’est le [`splat operator`](https://ruby-doc.com/core/doc/syntax/calling_methods_rdoc.html#label-Array+to+Arguments+Conversion).